testdata/networking/multus-cni/Pods/IPv6/ipv6-pod.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test-pod
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      name: test-pod
+  template:
+    metadata:
+      annotations:
+        k8s.v1.cni.cncf.io/networks: whereabouts-excludeipv6
+      labels:
+        name: test-pod
+    spec:
+      containers:
+      - name: test-pod
+        image: quay.io/openshifttest/hello-sdn@sha256:d5785550cf77b7932b090fcd1a2625472912fb3189d5973f177a5a2c347a1f95
+        ports:
+        - containerPort: 8080
+        - containerPort: 443
+        env:
+        - name: RESPONSE
+          value: test-pod
```

Comment:
**Request:** Yaml-Lint would want us to have a newline character at the end here. 
![image](https://user-images.githubusercontent.com/8180469/105376188-7b520080-5bd7-11eb-9f72-19a020848f1f.png) this symbol tells us that there is no newline character at the end of file

Reply:
Good catch, fixed it. Thanks!